spk/python310/src/requirements-crossenv.txt

```diff
@@ -22,6 +22,7 @@ immutables==0.19
 MarkupSafe==2.1.1
 msgpack-python==0.5.6
 netifaces==0.11.0
+psutil==5.9.4
```

Comment:
@th0ma7 is psutil added by intention?
_just to be sure_

Reply:
Nope, just reordered.  The sole intent is to facilitate wheel building testing when uncommenting requirements-crossenv.txt and serves as examples.

But by default crossenv wheels are always disabled and not packaged with py3.1*.  Only a minimal set of pure wheels are included.